logbook-servlet/src/main/java/org/zalando/logbook/servlet/RemoteRequest.java

```diff
@@ -82,22 +84,39 @@ public String getQuery() {
         return builder.build();
     }
 
+    @SuppressWarnings("unchecked") // warnings appear when using Servlet API 2.5
+    @Override
+    public Map<String, String[]> getParameterMap() {
+        Map<String, String[]> parameterMap = super.getParameterMap();
+        if (getContentType() == null || !getContentType().toLowerCase().startsWith("application/x-www-form-urlencoded")) {
+            return parameterMap;
+        }
+        String formEncoded = parameterMap.entrySet().stream()
+                .map(entry -> entry.getKey() + "=" + join(",", entry.getValue()))
```

Comment:
Is separation by comma correct here? I think it should be multiple key=value pairs instead.